### PR TITLE
MixtureModelPipeline that lets you specify a cluster to be tagged…

### PR DESCRIPTION
**Pull request by mamikonyana**
… as outliers

Also has python scripts that automate diagnostic process.

### CODE REVIEW

File: src/main/java/macrobase/analysis/classify/MixtureGroupClassifier.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        for (int i = 0; i < K; i++) {
+            double dist = clusters.get(i).getDistance(targetLocation);
+            if (dist < distanceToClosestCluster) {
+                distanceToClosestCluster = dist;
+                targetClusterIndex = i;
+            }
+        }
+        log.debug("target cluster index is {}", targetClusterIndex);
+        for (Datum d : records) {
+            //double[] probas = mixtureModel.getClusterProbabilities(d);
```
Comment (pbailis) [inline src/main/java/macrobase/analysis/classify/MixtureGroupClassifier.java]: remove?

File: src/main/java/macrobase/analysis/classify/MixtureGroupClassifier.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import macrobase.conf.MacroBaseConf;
+import macrobase.datamodel.Datum;
+import org.apache.commons.math3.linear.ArrayRealVector;
+import org.apache.commons.math3.linear.RealVector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class MixtureGroupClassifier implements OutlierClassifier {
```
Comment (pbailis) [inline src/main/java/macrobase/analysis/classify/MixtureGroupClassifier.java]: Comment on behavior would be great.

**Comment by pbailis** [comment]
Looks fine to me, save for the coverage...

**Comment by mamikonyana** [comment]
Added comments and improved coverage

**Pull request closed without merging**